docs/GraFx-Studio/convert/Adobe-InDesign/index.md

```diff
@@ -0,0 +1,196 @@
+|                        | Image frame with stretched content       | ✅             |                                                    |
+| **Primitives**         | Rectangle, ellipse, triangle             | ✅             |                                                    |
+|                        | Stroke weight                            | ✅             |                                                    |
+|                        | Stroke color (predefined and custom)     | ✅             |                                                    |
+|                        | Fill color (predefined and custom)       | ✅             |                                                    |
+|                        | Rectangle corner radius                  | ✅             |                                                    |
+|                        | Triangle corner radius                   | ❌ |                                    |
+|                        | Polygon stroke weight                    | ❌  |                                    |
+| **Predefined Color Export** | CMYK swatch colors| ✅           |                                                    |
+|  | RGB swatch colors| ✅           |                                                    |
+|  | Spot color swatch colors| ✅           |                                                    |
+| **Preflight**          | List of potential problems               | ✅             |                                                    |
+|                        | Details (click on an item in the list)   | ✅             |                                                    |
+|                        | Center problem object (click on warning) | ✅             |                                                    |
+|                        | No document open behavior                | ✅             | Preflight does nothing if no document is open     |
+|                        | Switching documents                      | ✅             | Preflight results reset when switching documents  |
+| **Layers**             | Layer names                              | ✅             | InDesign layer names converted to default names   |
+|                        | Hidden layers                            | ❌           | Hidden layers are not exported                    |
+
+### Legend (not John)
+
+- ✅ **Supported**: Fully supported
+- ⚠️ **Limited Support**: Partially supported; some functionality may be restricted
+- ❌ **Unsupported**: The team is working hard to add this functionality
+
+## Tips for Successful Conversion
+
+1. **Simplify Your Design**:
+   - Ensure your design uses basic text, shapes, and images for best results.
+
+2. **Check for Unsupported Features**:
+   - Use the **Preflight** option in InDesign to identify unsupported features before exporting.
+
+3. **Use Linked Assets**:
+   - Make sure all images and fonts are properly linked and included in the export.
```

Comment:
Font Files are not included in the .zip and are not exported.
User should have all Fonts installed to the platform. 
Fonts will be applied to the template upon Importing.
Otherwise the default font is used. (still User can Uppload required font at any moment after .zip is imported)